### PR TITLE
Added SQL commands related to doctrine and remove Doctrine usage

### PR DESCRIPTION
**Pull request by M0rgan01**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Added SQL commands related to doctrine, removing the execution of the command 'doctrine:schema:update'
| Type?             | improvement
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes https://github.com/PrestaShop/PrestaShop/issues/33874, Fixes https://github.com/PrestaShop/PrestaShop/issues/18113
| Sponsor company   | -
| How to test?      | Use https://github.com/PrestaShop/SeamlessUpgradeToolbox/tree/main/autoupgrade-scripts for test versions concerned. The SQL dump must be equivalent to a fresh install

### CODE REVIEW

**Comment by Hlavtox** [comment]
Guys what about this PR?

File: upgrade/sql/1.7.8.0.sql [hunk 1]
```diff
@@ -199,3 +199,11 @@ CREATE TABLE IF NOT EXISTS `PREFIX_feature_flag` (
 INSERT INTO `PREFIX_feature_flag` (`name`, `state`, `label_wording`, `label_domain`, `description_wording`, `description_domain`)
 VALUES
 	('product_page_v2', 0, 'Experimental product page', 'Admin.Advparameters.Feature', 'This page benefits from increased performance and includes new features such as a new combination management system. Please note this is a work in progress and some features are not available yet.', 'Admin.Advparameters.Help');
+
+DROP INDEX id_shop ON `PREFIX_shop_url`;
+DROP INDEX full_shop_url ON `PREFIX_shop_url`;
+DROP INDEX full_shop_url_ssl ON `PREFIX_shop_url`;
+ALTER TABLE `PREFIX_shop_url` CHANGE id_shop_url id_shop_url INT AUTO_INCREMENT NOT NULL, CHANGE id_shop id_shop INT NOT NULL;
+CREATE INDEX IDX_279F19DA274A50A0 ON `PREFIX_shop_url` (id_shop);
```
Comment (Quetzacoalt91) [inline upgrade/sql/1.7.8.0.sql:207]: Surprisingly, key names seem to be the same between two executions of the same command.

```
mysql> DESCRIBE ps_shop_url;
+--------------+--------------+------+-----+---------+----------------+
| Field        | Type         | Null | Key | Default | Extra          |
+--------------+--------------+------+-----+---------+----------------+
| id_shop_url  | int(11)      | NO   | PRI | NULL    | auto_increment |
| id_shop      | int(11)      | NO   | MUL | NULL    |                |
| domain       | varchar(150) | NO   |     | NULL    |                |
| domain_ssl   | varchar(150) | NO   |     | NULL    |                |
| physical_uri | varchar(64)  | NO   |     | NULL    |                |
| virtual_uri  | varchar(64)  | NO   |     | NULL    |                |
| main         | tinyint(1)   | NO   |     | NULL    |                |
| active       | tinyint(1)   | NO   |     | NULL    |                |
+--------------+--------------+------+-----+---------+----------------+
8 rows in set (0.00 sec)

mysql> SHOW INDEX FROM ps_shop_url;
+-------------+------------+----------------------+--------------+-------------+-----------+-------------+----------+--------+------+------------+---------+---------------+
| Table       | Non_unique | Key_name             | Seq_in_index | Column_name | Collation | Cardinality | Sub_part | Packed | Null | Index_type | Comment | Index_comment |
+-------------+------------+----------------------+--------------+-------------+-----------+-------------+----------+--------+------+------------+---------+---------------+
| ps_shop_url |          0 | PRIMARY              |            1 | id_shop_url | A         |           0 |     NULL | NULL   |      | BTREE      |         |               |
| ps_shop_url |          1 | IDX_279F19DA274A50A0 |            1 | id_shop     | A         |           0 |     NULL | NULL   |      | BTREE      |         |               |
+-------------+------------+----------------------+--------------+-------------+-----------+-------------+----------+--------+------+------------+---------+---------------+
2 rows in set (0.00 sec)
```

**Review by Quetzacoalt91 — APPROVED** [review]

**Review by kpodemski — APPROVED** [review]
Awesome! I still think we could bring back Doctinre at some point, but this change now should save a lot of trouble during the upgrades! well done @M0rgan01

**Review by Quetzacoalt91 — APPROVED** [review]

**Review by AureRita — APPROVED** [review]
Hi @M0rgan01 

Thank you for your PR, I tested it : 

From 1.7.0.1 to : 
- 1.7.1..0
- 1.7.1.1
- 1.7.2.0
- 1.7.3.0
- 1.7.4.0
- 1.7.6.0
- 1.7.7.0
- 1.7.8.11

From 1.7.5.1 to : 
- 8.0.5
- 8.1.5

From 8.0.5 to 9.0.0 

Because all seems to be corrected as you said, It's QA :heavy_check_mark: 

Thank you

**Pull request merged**